skimage/restoration/_denoise.py

```diff
@@ -610,11 +669,14 @@ def denoise_wavelet(image, sigma=None, wavelet='db1', mode='soft',
     image, but larger thresholds also decrease the detail present in the image.
 
     If the input is 3D, this function performs wavelet denoising on each color
-    plane separately. The output image is clipped between either [-1, 1] and
-    [0, 1] depending on the input image range.
+    plane separately.
 
-    When YCbCr conversion is done, every color channel is scaled between 0
-    and 1, and `sigma` values are applied to these scaled color channels.
+    For floating point inputs, the original input range is maintained and there
+    is no clipping applied to the output. Other input types will be converted
+    to a floating point value in the range [-1, 1] or [0, 1] depending on the
+    input image range. Any rescaling applied to the `image` will also be
+    applied to `sigma` to maintain the same relative amplitude for the supplied
+    noise standard deviation.
```

Comment:
@grlee77 Sorry, this just occurred to me. Could you add a heading to this section, `.. versionchanged:: 0.16`? This makes it easier for people scouring the docs for reasons why their output has changed. Do `git grep versionchanged` in our repo for example usages. It's helped me a few times with NumPy changes.

Reply:
I agree it can be useful. I have added it now.